File: guanabara-python/src/lista02/exercicio17.py
velocidade = int(input("Insira a velocidade do veículo: "))
velocidade_excedente = velocidade - 80
multa = velocidade_excedente * 5

if velocidade > 80:
    print(f"Velocidade: {velocidade}. Usuário multado! \n Valor da multa: {multa}")
else:
    print(f"{velocidade}")

File: guanabara-c/lista02/exercicio17.c
#include <stdio.h>

int main(){

    int velocidade, velocidade_excedente, multa;

    printf("Insira a velocidade do veículo: ");
    scanf("%d", &velocidade);

    if(velocidade > 80){
        
        velocidade_excedente = velocidade - 80;
        multa = velocidade_excedente * 5;
        printf("O valor da multa é de R$ %d", multa);
    }

    return 0;
}
